src/lib/common/tag.h

```diff
@@ -30,6 +30,22 @@
 #include "common/Format.h"
 
 
+// Macros to be used for JSON rendering
+
+#define JSON_PROP(name) std::string("\"" + std::string(name) + "\":")
+
+#define JSON_STR_LAST(value) std::string("\"" + std::string(value) + "\"")
+#define JSON_STR(value) std::string(JSON_STR_LAST(value) + ",")
+
+#define JSON_NUMBER_LAST(value) std::string(value)
```

Comment:
This here will probably not work (JSON_NUMBER*).
It would be great if it did but ... 'value' is supposed to be an integer and we'd need a constructor of std::string accepting integers.
I don't think there is one (not that I have investigated it too much).

I normally end up doing something like this:

    int x = 12;
    char cV[12];

    snprintf(cV, sizeof(cV), "%d", x);